kong/plugins/prometheus/api.lua

```diff
@@ -1,11 +1,11 @@
 local exporter = require "kong.plugins.prometheus.exporter"
 
-local printable_metric_data = function()
+local printable_metric_data = function(_)
```

Comment:
Is param `_` necessary?

Reply:
This is a hint that we are skipping a parameter. This function is a call back which is called with the parameter "data".